### ISSUE TITLE
Bad parsing in module-level comments when there is /* in a verbatim string

### ISSUE THREAD

**Issue by federicomenaquintero** [rust-lang/rust#157034]
I tried this code:

```rust
/*!
When processing an SVG, librsvg will only load referenced files if
they are in the same directory as the base file, or in a
subdirectory of it.  That is, if the base file is
`/foo/bar/baz.svg`, then librsvg will only try to load referenced
files (from SVG's `<image>` element, for example, or from content
included through XML entities) if those files are in `/foo/bar/*`
or in `/foo/bar/*/.../*`.  This is so that malicious SVG documents
cannot include files that are in a directory above.
*/

fn main() {
    todo!()
}
```

playground link - https://play.rust-lang.org/?version=nightly&mode=debug&edition=2024&gist=304576253624fa7fdaad266ae7d98a3c

I expected to see this happen: no compilation errors.

Instead, this happened: The code above is a shortened section of `librsvg/rsvg/src/lib.rs`.  I get this compilation error:

```
error[E0758]: unterminated block doc-comment
   --> rsvg/src/lib.rs:1:1
    |
  1 |   /*!
    |   ^-
    |   |
    |  _unterminated block doc-comment
    | |
  2 | | Load and render SVG images into Cairo surfaces.
  3 | |
  4 | | This crate can load SVG images and render them to Cairo surfaces,
...   |
 91 | | or in `/foo/bar/*/.../*`.  This is so that malicious SVG documents
    | |                      -- ...as last nested comment starts here, maybe you want to close this instead?
...   |
140 | | */    
    | | -- ...and last nested comment terminates here.
...   |
275 | |     pub use crate::surface_utils::shared_surface::{SharedImageSurface, SurfaceType};
276 | | }
    | |__^
```

I think the parser is getting confused about `/foo/bar/*/.../*` and thinks that the `/*` in there are comment delimiters, when they are in fact part of the verbatim string.

(... apologies if I'm getting my syntax completely wrong)

### Meta

`rustc --version --verbose`:
```
rustc 1.92.0 (ded5c06cf 2025-12-08)
binary: rustc
commit-hash: ded5c06cf21d2b93bffd5d884aa6e96934ee4234
commit-date: 2025-12-08
host: x86_64-unknown-linux-gnu
release: 1.92.0
LLVM version: 21.1.3
```

This happens in the nightly playground, too.

</p>
</details>

**Comment by bjorn3** [comment]
It is expected that rustc does not do any parsing of comment contents other than for determining block comment nesting depth.
